docs/usage/dependency-pinning.md

```diff
@@ -118,6 +118,8 @@ With this approach, updates will be essentially "silent" - causing no notificati
 
 ### Scheduling
 
+TODO Discuss what to do with the scheduling section in the deep dive about pinning javascript deps.
```

Comment:
@rarkins Checking if I understand correctly: keep this section as it is now, but add a link to the new key `concepts/scheduling` page?

Reply:
No need for scheduling to be in the FAQ if it's featured prominently in a key concepts. I think it can be removed

Reply:
Okay so drop scheduling from the FAQ (I already did that, but just to confirm.)

What do we want with this section: https://docs.renovatebot.com/dependency-pinning/#scheduling that talks about scheduling within the "deep dive on JavaScript dependency pinning". We now have `stabilityDays` and later on Merge Confidence to do this job.

Reply:
I'm happy to leave that pinning piece as-is, although you're welcome to add a link to your new doc

Reply:
> I'm happy to leave that pinning piece as-is, although you're welcome to add a link to your new doc

I'll leave it as-is, but add a link to my new doc.